src/adapter/src/command.rs

```diff
@@ -424,7 +602,19 @@ pub enum SimpleResult {
         col_names: Vec<String>,
     },
     /// The query executed successfully but did not return rows.
-    Ok,
+    Ok {
+        ok: Option<String>,
```

Comment:
Thoughts on having this be `Ok { status: String, message: Option<String> }`? Otherwise consumers of this will have to explicitly check all of these fields.

Reply:
Ah, yeah--better idea.